requirements.in

```diff
@@ -1,7 +1,7 @@
 #-----------------------------------
 #   FSD Utils
 #-----------------------------------
-git+https://github.com/communitiesuk/funding-service-design-utils.git@main
+git+https://github.com/communitiesuk/funding-service-design-utils.git@fs-1169-common-config
```

Comment:
Can we switch to use a released version please?

Reply:
Yes - once the PR on fsd_utils has been merged it will create a new version and I'll update this to use it: https://github.com/communitiesuk/funding-service-design-utils/pull/10